content/applications/inventory_and_mrp/plm/manage_changes/eco_type.rst

```diff
@@ -39,105 +49,86 @@ the following information:
 
 Stages
 ======
 
-Within an |ECO| type project, *stages* are like milestones and are used to identify the progress of
-the |ECO| before the changes are ready to be applied. (e.g. 'Feedback', 'In Progress', 'Approved',
-'Complete')
-
-Additionally, required approvers can be added to each stage, ensuring that changes to the production
-|BOM| cannot proceed until the approver reviews and approves the |ECO|. Doing so prevents errors on
-the production |BOM| by enforcing at least one review of suggested changes before they're applied on
-a production |BOM|.
+Within the :guilabel:`Engineering Change Orders` Kanban view for a specific |ECO| type, *stages* are
+milestones used to identify the progress of the |ECO| before the changes are ready to be applied. By
+default, Odoo uses `New`, `In-Progress`, `Validated`, and `Effective`, but these are fully
+customizable to the specific life cycle of an |ECO| type.
 
-For best practice, there should be at least one *verification* stage, which is a stage with a
-required approver, and one *closing* stage, which stores |ECOs| that have been either cancelled or
-approved for use as the next production |BOM|.
+.. image:: eco_type/eco-stage-defaults.png
+   :alt: Default stages for an ECO type.
 
-Create stage
-------------
+.. note::
+   The `Effective` stage is folded by default to avoid listing every single |ECO| that has gone into
+   effect. See :ref:`Closing stage<plm/eco/closing-stage>` for more on this configuration.
 
-To add a stage, go to the :menuselection:`PLM` app and select the intended project for an |ECO| type
-from the :guilabel:`PLM Overview` dashboard.
+:ref:`Verification stages <plm/eco/verification-stage>` require a designated user to approve,
+ensuring that changes do not proceed until the |ECO| changes have been reviewed by the appropriate
+stake holders. :ref:`Closing stages<plm/eco/closing-stage>` that can apply changes put |BOM| and
+operations changes into immediate effect, changing all pending and future :abbr:`MO (Manufacturing
+orders)` to the latest |BOM| version.
 
-Then, on the :guilabel:`Engineering Change Orders` project pipeline for the |ECO| type, click the
-:guilabel:`+ Stage` button. Doing so reveals a text box to fill in the name of the stage. After
-filling it in, click the :guilabel:`Add` button to finish adding the stage.
+.. tip::
+   The most common practice is to have at least one *verification* stage, which is a stage with a
+   required approver, and one *closing* stage, which stores |ECOs| that have been either cancelled
+   or approved for use as the next production |BOM|.
 
-.. example::
-   A new `Assigned` stage separates assigned |ECOs| from the unassigned ones in the `New` stage.
-   Adding another stage helps the product manager track unassigned tasks.
+.. seealso::
+   :doc:`Approvals<../management/approvals>`
 
-   .. image:: eco_type/create-stage.png
-      :align: center
-      :alt: Create a new stage in a project for an ECO type.
```

Comment:
Please make sure to also remove this unused image from the folder (`eco_type/create-stage.png`)